### PR TITLE
Improvement/mapping proxy type

### PR DESCRIPTION
**Pull request by sudiptob2**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.idea
```
Comment (0hsn) [inline .gitignore:130]: please add a [git global ignore](https://stackoverflow.com/questions/7335420/global-git-ignore) and add it there.

File: chk/modules/http/request_helper.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
 """
 Http module helpers
 """
+from types import MappingProxyType
+from urllib.parse import unquote, urlparse
+
+from requests import request, Response
+from requests.auth import HTTPBasicAuth
+
 from chk.infrastructure.work import RequestProcessorContract
```
Comment (0hsn) [inline chk/modules/http/request_helper.py:10]: - organize by alphabetic order
- remove unnecessary new line

```suggestion
from types import MappingProxyType
from urllib.parse import unquote, urlparse
from requests import request, Response
from requests.auth import HTTPBasicAuth
from chk.infrastructure.work import RequestProcessorContract
```

**Review by 0hsn — COMMENTED** [review]
looks good. just improve those. thanks.

File: .gitignore (hunk 1 above)
Comment (sudiptob2) [reply .gitignore:130]: thanks will update

File: chk/modules/http/request_helper.py (hunk 2 above)
Comment (sudiptob2) [reply chk/modules/http/request_helper.py:10]: According to the python [PEP8](https://peps.python.org/pep-0008/#imports) guideline, I think we should put a blank line between each group of imports. @0hsn

Comment (0hsn) [reply chk/modules/http/request_helper.py]: fair enough

**Comment by 0hsn** [comment]
@sudiptob2 also please remove it from
- [ ] Pipfile
- [ ] requirements.txt
- [ ] run `pipenv install` so that Pipfile.lock gets updated

**Comment by sudiptob2** [comment]
> * [ ]  Pipfile
> *

`DotMap` is used in other places too. It has one use in `app_container.py`

**Comment by 0hsn** [comment]
@sudiptob2 let's just use `dict` instead of `mappingproxytype` for `app_container.py` only (if necessary, or leave dynamic for now)

**Review by 0hsn — APPROVED** [review]

**Pull request merged**